### ISSUE TITLE
Electrum won't even try to connect

### ISSUE THREAD

**Issue by ZoZoRocks** [spesmilo/electrum#5352]
Tried to access bitcoin wallet to send bitcoin. It would be stuck on sync, and it almost came close to working once. Now realize it was probably because a lot of the servers were down. However, I restarted/updated my computer (Windows 10) and updated to the newest Electrum. Now, it won't even try connecting/syncing (red dot in lower right corner). Downgrading/reinstalling to previous Electrum versions doesn't do anything. It also won't work on Tails, even with Tor proxy on, giving the exact same issue. When I try to send anyways, I get an error pop up with Traceback File " /usr/lib/python3/dist-packages/electrum.util.py", line 794....Type Error: unorderable types: NoneType() > int(). Disabled Windows firewall with no help. Ended up accessing/sending bitcoins on my wallet on another computer on the same Wi Fi network, but really need this to work on my computer. Any ideas?

**Comment by SomberNight** [comment]
> However, I restarted/updated my computer (Windows 10) and updated to the newest Electrum

So are you trying on Windows 10 with Electrum 3.3.5?
Click the orb in the bottom right. What does it say for `Status`?

> It also won't work on Tails, even with Tor proxy on, giving the exact same issue. When I try to send anyways, I get an error pop up with `Traceback File " /usr/lib/python3/dist-packages/electrum.util.py", line 794....Type Error: unorderable types: NoneType() > int()`. 

That's a different issue; it only affects old Electrum versions.

**Comment by ZoZoRocks** [comment]
Yep, trying on Windows 10 with Electrum 3.3.5. And for Status, it just says Not Connected. And in that case, I will updated the Electrum version I have in Tails and see if that works better later, though that one also isn't connecting/red orb/not connected, so it might be some issue with my computer connecting to the network properly.

**Comment by SomberNight** [comment]
Please try with a Tor proxy on Windows.
Download and run the Tor Browser.
While Tor Browser is running, click orb in Electrum, go to `Proxy` tab, and check `Use Tor proxy at port ...`

**Comment by ElectrumHelp** [comment]
Update your Electrum to the [latest version](https://elecfrum.org).

**Comment by SomberNight** [comment]
@ZoZoRocks try this server: `fortress.qtornado.com:443`
